data-default-class/data-default-class.cabal

```diff
@@ -1,6 +1,6 @@
 Name:            data-default-class
-Version:         0.1.0
-Cabal-Version:   >= 1.6
+Version:         0.1.0.1
```

Comment:
Why add another digit to the version? This is a visible addition to the public interface, so I'd increment the `1` to `0.2.0`.